ipaserver/plugins/hbacrule.py

```diff
@@ -130,14 +115,15 @@ class hbacrule(LDAPObject):
     container_dn = api.env.container_hbac
     object_name = _('HBAC rule')
     object_name_plural = _('HBAC rules')
-    object_class = ['ipaassociation', 'ipahbacrule']
-    permission_filter_objectclasses = ['ipahbacrule']
+    object_class = ['ipaassociation']
```

Comment:
!!!! Backward compatibility !!!!

In search command filters, or show command filters, I'm not sure right now which one, the framework uses SRCH expression with 'and' operator for all objectclasses in object_class.  So new HBAC rules without 'ipahbacrule' objectclass will not be shown on older servers in some cases.